src/Components/SearchResultsContainer/SearchResultsContainer.css

```diff
@@ -15,6 +14,7 @@
   border: 1px solid #c4c4c4;
   border-radius: 5px;
   padding: 2%;
+  visibility: hidden;
```

Comment:
I inadvertently introduced a bug with the original code. Because the search results container was hidden...it was impossible to see any search results. This change hides just the sort by functionality.